common/package.json

```diff
@@ -48,6 +48,7 @@
     "raven-js": "^3.26.3",
     "raw-loader": "^2.0.0",
     "react": "^16.13.1",
+    "react-color": "^2.18.1",
```

Comment:
https://bundlephobia.com/result?p=react-color@2.18.1 😱

Reply:
I did actually check bundlephobia but mistakenly thought that this PR https://github.com/casesandberg/react-color/pull/722 sorted things out - turns out it's still massive, but more important the maintainer has tagged but not released the version including that 🙃 . I will have a brief look at using a smaller library because you're right, this is obscene.